www/source/blog/2018-05-01-Hab-OSB.html.md

```diff
@@ -0,0 +1,40 @@
+
+For example, if you are running stateless microservices within a federated, cross-cloud Kubernetes cluster, and you wanted your application to be able to leverage a large distributed database service that you had running on bare metal, off cluster, for performance reasons, you could use the Open Service Broker to make that database available as part of a service catalog, and then have your applications connect to it from within the Kubernetes cluster -- or wherever they may be running -- with a simple command. 
+
+Habitat and the Open Service Broker are a great pairing in multiple dimensions. If you are using Habitat to automate the building and management of a catalog of distributed services for your development teams, Open Service Broker is an excellent way to do that, and it has been designed in partnership with popular enterprise platforms such as Kubernetes, CloudFoundry, and OpenShift. Continue to use Habitat to continuously build and ship your applications and services, run those services wherever fits your business best, and use Open Service Broker as the intelligent glue so that your consumers don’t need to worry about where and how the services they call are being run. 
+
+For example, a developer deploying applications to CloudFoundry -- either natively or built with Habitat -- can call out to a Habitat Open Service Broker maintained by her Operations team to easily request a service she needs. The Habitat Open Service Broker will receive that request, and fulfill it using a Habitat package in the topology the developer wants, on the infrastructure the Operations team has decided is the best fit (in our video demo below, this is a Kubernetes cluster, but it can be extended to the best infrastructure for the job). The developer’s application receives back the necessary service information to connect to and use the service she requested. 
+
+Additionally, you can follow the same mechanism to integrate your Habitat built applications with other Open Service Broker providers, such as Azure or Google’s, to do the same when there are cloud hosted services you would like to provide to your consumers. Open Service Broker is a great way to provide a rich array of services to your users.
+
+On Friday, awesome Kinvolk team member [Indradhanush Gupta](https://twitter.com/indradhanush92) showed how to use the [Habitat Open Service Broker](https://github.com/kinvolk/habitat-service-broker) that he and [Lili Cosic](https://twitter.com/LiliCosic) have been working on. It shows deploying services on a Kubernetes cluster via the OSB, but following this pattern you could deploy to other infrastructures as well using Habitat packages.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/VnW9UA77kIE" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+https://www.youtube.com/watch?v=VnW9UA77kIE&feature=youtu.be
```

Comment:
You can probably delete this line, since the video title itself is linked -- or if you do want to keep it, Markdownify it like so:

[https://www.youtube.com/watch?v=VnW9UA77kIE&feature=youtu.be](https://www.youtube.com/watch?v=VnW9UA77kIE&feature=youtu.be)